research/src/pages/skeleton.proposal.mdx

```diff
@@ -0,0 +1,53 @@
+---
+menu: Components
+name: Skeleton (Editor's Draft)
+path: /components/skeleton
+pathToResearch: /components/skeleton.research
+---
+
+## Overview
+
+Displays a placeholder reserving the place for the content that is being loaded or used to prototype layouts.
+
+## Use Cases
+
+The Skeleton component is primarily leveraged to reserve a place in the layout for a content that will be shown asynchronous avoiding elements moving to adapt to new dimensions.
+
+## Prior Art/Examples <a href="#prior-art" id="prior-art"></a>
+
+- [Ant Design](https://ant.design/components/skeleton/)
+- [Semantic UI](https://semantic-ui.com/elements/placeholder.html)
+- [Fabric UI](https://developer.microsoft.com/en-us/fluentui#/controls/web/shimmer)
+
+## &lt;skeleton&gt; Properties <a href="#skeleton-properties" id="skeleton-properties"></a>
+
+| Attribute Name | Type     | Default Value | Description                                                                                |     |
+| -------------- | -------- | ------------- | ------------------------------------------------------------------------------------------ | --- |
+| `fluid`        | `bool`   | `false`       | A fluid Skeleton fills up the entire width of its container.                               |     |
+| `animation`    | `string` | `off`         | Define the animation for the Skeleton, possible "wave", "pulse" or "off" for no animation. |     |
+
+## &lt;skeleton-line&gt; Properties <a href="#skeleton-line-properties" id="skeleton-line-properties"></a>
+
+| Attribute Name | Type     | Default Value | Description                             |     |
+| -------------- | -------- | ------------- | --------------------------------------- | --- |
+| `height`       | `string` | `100%`        | The height the line should fill.        |     |
+| `width`        | `string` | `100%`        | The width the line should fill.         |     |
+| `round`        | `bool`   | `false`       | Set whether should have rounded borders |     |
+
+## &lt;skeleton-shape&gt; Properties <a href="#skeleton-shape-properties" id="skeleton-shape-properties"></a>
+
+| Attribute Name | Type     | Default Value | Description                            |     |
+| -------------- | -------- | ------------- | -------------------------------------- | --- |
+| `height`       | `string` | `0px`         | The height the line should fill.       |     |
+| `width`        | `string` | `0px`         | The width the line should fill.        |     |
+| `round`        | `bool`   | `false`       | Define if the shape should be a circle |     |
+
+#### Structure <a class="link" href="#structure" id="structure"></a>
+
+- `<skeleton>` - The root element that contains the lines or shapes and controls the internal animation **[required]**
+- `<skeleton-line>` - retangular fluid element representing a line
+- `<skeleton-shape>` - element representing a shape
+
+### Content not allowed within the anatomy
```

Comment:
I would love to see the aria information in this proposal that we received from the ARIA WG on making an accessible skeleton component

Reply:
which information is that?

Reply:
@assuncaocharles sorry for the delay - here it is: https://github.com/w3c/aria/issues/1317#issuecomment-687346933